gateway/Cargo.toml

```diff
@@ -19,10 +20,14 @@ tokio-tungstenite = "0.17"
 toml = "0.5.9"
 uuid = "1.1.0"
 
-gateway-messages = { path = "../gateway-messages", features = ["std"] }
-gateway-sp-comms = { path = "../gateway-sp-comms" }
 omicron-common = { path = "../common" }
 
+[dependencies.gateway-messages]
```

Comment:
Seems somewhat dangerous to do this, but I figure you did it on purpose since MGS isn't running anywhere yet, and it's under heavy development.

Reply:
comment withdrawn :)

Reply:
chatted offline - pinned to a specific commit in cb0f6b1